### ISSUE TITLE
Endpoints /api/accounts/open and /api/accounts/open2 not working

### ISSUE THREAD

**Issue by a1300** [AschPlatform/asch-core#18]
### Version
__asch-core__ version 1.4.0
__asch__ version 1.4.0

### Environment
Kubuntu Linux 4.15.0-29-generic

#### 1 Start ASCH node
```bash
node app.js
```
#### 2 Start terminal and execute call endpoint with `curl`
__open__
```bash
curl -X POST -H "Content-Type: application/json" -k -d '{"secret":"sentence weasel match weather apple onion release keen lens deal fruit matrix"}' http://localhost:4096/api/accounts/open

# return value:  
{
  "success":false,
  "error":"Missing required property: address"
}
```

__open2__
```bash
curl -X POST -H "Content-Type: application/json" -k -d '{"publicKey":"a7cfd49d25ce247568d39b17fca221d9b2ff8402a9f6eb6346d2291a5c81374c"}' http://localhost:4096/api/accounts/open2

# return value:  
{
  "success":false,
  "error":"Missing required property: address"
}
```

### What is expected?

__open__
```js
{   
    "success": true,    
    "account": {   
        "address": "AHMCKebuL2nRYDgszf9J2KjVZzAw95WUyB",    
       ...
       ...
    }   
}   
```

__open2__
```js
{   
	"success": true,   
	"account": {   
		"address": "AHMCKebuL2nRYDgszf9J2KjVZzAw95WUyB",  
               .... 
	},   
	"latestBlock": {   
             ....
	},   
	"version": {   
              ....
	}
}
```

### What is actually happening?

Error message for __open__ and __open2__:  
```json
{
  "success":false,
  "error":"Missing required property: address"
}
```

**Comment by bassjobsen** [comment]
@a1300 i believe that my pull request should fix your issues

**Comment by sqfasd** [comment]
already fixed in 60bc4f80
